<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_DisplaySongTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>891</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_AddSong">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MOpenSong.png</normaloff>:/new/prefix1/images/MOpenSong.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PlaySong">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MPlay.png</normaloff>:/new/prefix1/images/MPlay.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PreviousSong">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MPrevious.png</normaloff>:/new/prefix1/images/MPrevious.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PauseSong">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MPause.png</normaloff>:/new/prefix1/images/MPause.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_StopPlaySong">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MStops.png</normaloff>:/new/prefix1/images/MStops.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PlayNextSong">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MNext.png</normaloff>:/new/prefix1/images/MNext.png</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_PlayProgress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>741</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_Volume">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SoundYesNo">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MSound.png</normaloff>:/new/prefix1/images/MSound.png</iconset>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_PlayTime">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,0)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>891</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>在线音乐列表</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_SongList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>871</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255,70)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>891</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_St">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,70)</string>
     </property>
     <property name="text">
      <string>动手搜一搜</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MDriect.png</normaloff>:/new/prefix1/images/MDriect.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InputSongs">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255,70)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SearchSong">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Search...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/prefix1/images/MSerach.png</normaloff>:/new/prefix1/images/MSerach.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_About">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/new/prefix1/images/MAbout.png</normaloff>:/new/prefix1/images/MAbout.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/new/prefix1/images/MQuit.png</normaloff>:/new/prefix1/images/MQuit.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
